cmds/clients/contestcli-http/main.go

```diff
@@ -149,12 +183,40 @@ func run(verb string) error {
 		encoder.SetIndent("", " ")
 		err := encoder.Encode(apiErr)
 		if err != nil {
-			return fmt.Errorf("cannot re-encode httplistener.HTTPAPIResponse object: %v", err)
+			return "", fmt.Errorf("cannot re-encode httplistener.HTTPAPIResponse object: %v", err)
 		}
 		indentedJSON = buffer.Bytes()
 	}
 	// this is the only thing we want on stdout - the JSON-formatted response,
 	// so it can be piped to other tools if desired.
-	fmt.Println(string(indentedJSON))
-	return nil
+	return string(indentedJSON), nil
 }
+ 
+func wait(params url.Values) (string, error) {
```

Comment:
could you make the poll interval a parameter for this function? It will make it easier to test

Reply:
Done